### PR TITLE
Upgrade azure sdk

### PR DESCRIPTION
**Pull request by rintcius**
Upgrade Azure SDK which includes some token expiration fixes.

Still WIP - need to remove deprecation warnings & make sure we use the same Netty version across all repositories.

### CODE REVIEW

**Comment by rintcius** [comment]
[ch12535]

File: build.sbt [hunk 1]
```diff
@@ -17,7 +17,7 @@ val AwsSdkVersion = "2.9.1"
 val Fs2Version = "2.4.5"
 val MonixVersion = "3.3.0"
 // Make sure this is the same for different sub projects
-val NettyVersion = "4.1.49.Final"
+val NettyVersion = "4.1.52.Final"
```
Comment (wemrysi) [inline build.sbt]: Is this just documentation? It isn't used anywhere I can see.

File: azure/src/test/scala/quasar/blobstore/azure/AzurePropsServiceSpec.scala [hunk 2]
```diff
@@ -91,12 +91,5 @@ class AzurePropsServiceSpec extends Specification with CatsIO {
         BlobPath(List(PathElem("something"))),
         beNone)
     }
-
-    "invalid container returns none" >> {
```
Comment (wemrysi) [inline azure/src/test/scala/quasar/blobstore/azure/AzurePropsServiceSpec.scala:95]: Why was this and the other test removed?

File: build.sbt (hunk 1 above)
Comment (rintcius) [reply build.sbt]: Ah yes, I noticed this too, was just busy removing this

File: azure/src/test/scala/quasar/blobstore/azure/AzurePropsServiceSpec.scala (hunk 2 above)
Comment (rintcius) [reply azure/src/test/scala/quasar/blobstore/azure/AzurePropsServiceSpec.scala:95]: The service cannot be created from this config anymore. 
Was planning to replace them with something else before merge though, if possible.

**Comment by rintcius** [comment]
Note that there are no recent AWS and Azure SDK versions that are built with the same Netty version.
The new AWS SDK version uses 4.1.53 and Azure SDK version 4.1.52. Plan is to set the version 4.1.53 in quasar now and test if that's going ok.

**Comment by rintcius** [comment]
snapshot `4.0.5-6ad35e0`

**Review by wemrysi — APPROVED** [review]

**Pull request merged**